### PR TITLE
Update npm config to fix dependency issues.

### PR DESCRIPTION
**Pull request by eschlange**
* Updated versions of cordova.
* Specify localnotification version.
* Removing config token logic until more session logic is complete.
* Removing unused cordova plugin files.

### CODE REVIEW

**Comment by eschlange** [comment]
@ericcf @michaelwehrley Please take a quick look at these config updates. I am blocked until these are merged.

File: src/js/Application.js [hunk 1]
```diff
@@ -125,16 +125,12 @@
       }
 
       settings.fetch().then(function() {
-        if(!authenticationTokenCache.first()) {
-          $location.url(Routes.CONFIG);
-        } else {
+        determineRouteFromState();
```
Comment (ericcf) [inline src/js/Application.js]: Did you mean to bundle application logic updates into this commit related to dependency updates?

**Comment by michaelwehrley** [comment]
Besides @ericcf's concern, this looks good to me. :smiling_imp:

**Comment by eschlange** [comment]
@ericcf @michaelwehrley I'm going to merge this in for now so that I don't muck up the commit history too much. If you would like me to strip the configuration token change and add it to a different pull request, let me know.

**Pull request merged**